lib/v2/sdu/extractor/sdrj.js

```diff
@@ -0,0 +1,19 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (link, ctx) =>
+    await ctx.cache.tryGet(link, async () => {
+        try {
+            const result = await got(link);
+            const $ = cheerio.load(result.data);
+            const content = $('#vsb_content').html();
+            const author = $("form[name='_newscontent_fromname'] > h1").text();
+            const exactDateLine = $("form[name='_newscontent_fromname'] > p.info").text().trim();
+            const exactDateText = exactDateLine.match(/^发布时间：(?<date>\d+\/\d+\/\d+\s\d{2}:\d{2}:\d{2})/).groups.date;
+            const exactDate = parseDate(exactDateText, 'YYYY/MM/DD HH:mm:ss');
```

Comment:
```suggestion
            const exactDate = timezone(parseDate(exactDateText, 'YYYY/MM/DD HH:mm:ss'), +8);
```
You also need to adjust timezone.
`const timezone = require('@/utils/timezone');`